src/main/java/net/minecraftforge/fml/common/FMLCommonHandler.java

```diff
@@ -202,7 +202,7 @@ public Side getSide()
     public Side getEffectiveSide()
     {
         Thread thr = Thread.currentThread();
-        if (thr.getName().equals("Server thread") || thr.getName().startsWith("Netty Server IO"))
+        if (thr.getName().equals("Server thread") || thr.getName().startsWith("Netty Epoll Server IO") || thr.getName().startsWith("Netty Server IO"))
```

Comment:
Mods can include dependencies which start threads with names containing "Server", or a mod can start a thread with a name containing "Server" itself. `getEffectiveSide` is *not* meant to cover all cases.